nerdtribe/mr-binge
# [#15] Feature/Core Add Database

## Explanation of work done and why:


This PR adds in core reading and writing "demo" style along with example tests.
Closes #15 

## How to (manually) test:

1. Fetch! `git fetch -p && git checkout -b origin/feature/core-add-database`
2. Serve `yarn electron:serve`
3. Enter a string in the API key input box
![image](https://user-images.githubusercontent.com/8315447/66016816-4ed81980-e4a6-11e9-80a9-d5e8a1729bc1.png)
4. Verify that the input box a) clears the input and b) outputs the api key below
![image](https://user-images.githubusercontent.com/8315447/66016852-6fa06f00-e4a6-11e9-868b-c0a456828c2a.png)
5. Reload the window and verify the last API Key is rendered the same as step 4.

## Checklist:
![image](https://user-images.githubusercontent.com/8315447/66016737-e5580b00-e4a5-11e9-9e33-e7fdc4ae8c58.png)
